azure-pipelines-integration-lsp.yml

```diff
@@ -0,0 +1,29 @@
+# Branches that trigger a build on commit
```

Comment:
Why did you choose to go with a separate queue versus just having another configuration on the existing one?

Reply:
(@dibarbet reminded me this was my own advice, so we can control CI gates separately on the GitHub UI)